Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedOperation.swift

```diff
@@ -0,0 +1,129 @@
+import Foundation
+import MongoSwiftSync
+
+struct UnifiedOperation: Decodable {
```

Comment:
I looked into trying to reuse the old type but there are some notable differences, e.g. here "object" can be _any_ string since it can an arbitrary entity name, there's now the option to save the result as an entity, etc.